frontend/src/app/projects/models/model-source/choose-source.component.html

```diff
@@ -2,48 +2,65 @@
+<div class="flex justify-center">
+  <mat-card class="mat-card-form" *ngIf="projectService.project$ | async">
+    <div
+      *ngIf="(projectService.project$ | async) && (modelService.model$ | async)"
+    >
+      <div
+        *ngIf="
+          (modelService.model$ | async)!.tool?.integrations?.t4c &&
+          userService.user?.role === 'administrator'
+        "
+      >
+        <h3>Git sources</h3>
+        <mat-divider></mat-divider>
+        <div class="my-[10px]">
+          <a mat-button (click)="modelSourceSelection.emit('git-add')">
+            <span> Use existing repository </span>
+            <mat-icon class="mat-icon-position right">navigate_next</mat-icon>
+          </a>
+        </div>
+      </div>
 
-  <div
-    *ngIf="
-      (modelService.model$ | async)!.tool?.integrations?.t4c &&
-      userService.user?.role === 'administrator'
-    "
-  >
-    <h2 class="title text-center">Team4Capella sources</h2>
-    <div class="flex flex-wrap justify-center">
-      <a (click)="modelSourceSelection.emit('t4c-existing')">
-        <mat-card class="option-card">
-          <p>Use existing repository</p>
-          <app-mat-icon size="50px">link</app-mat-icon>
-        </mat-card>
-      </a>
-      <a (click)="modelSourceSelection.emit('t4c-new')">
-        <mat-card class="option-card">
-          <p>Create new repository</p>
-          <app-mat-icon size="50px">add_circle_outline</app-mat-icon>
-        </mat-card>
-      </a>
-    </div>
-  </div>
+      <div
+        *ngIf="
+          (modelService.model$ | async)!.tool?.integrations?.t4c &&
+          userService.user?.role === 'administrator'
+        "
+      >
+        <h3>Team4Capella sources</h3>
+        <mat-divider></mat-divider>
+        <div class="my-[10px]">
+          <a mat-button (click)="modelSourceSelection.emit('t4c-existing')">
+            <span> Use existing repository </span>
+            <mat-icon class="mat-icon-position right">navigate_next</mat-icon>
+          </a>
+        </div>
+        <div
+          class="my-[10px]"
+          *ngIf="
+            (modelService.model$ | async)!.tool?.integrations?.t4c &&
+            userService.user?.role === 'administrator'
+          "
+        >
+          <a mat-button (click)="modelSourceSelection.emit('t4c-new')">
+            <span> Create new repository </span>
+            <mat-icon class="mat-icon-position right">navigate_next</mat-icon>
+          </a>
+        </div>
+      </div>
 
-  <h2 class="title text-center">Other</h2>
-  <div class="flex flex-wrap justify-center">
-    <a (click)="modelSourceSelection.emit('skip')">
-      <mat-card class="option-card">
-        <p>Skip this step</p>
-        <app-mat-icon size="50px">skip_next</app-mat-icon>
-      </mat-card>
-    </a>
-  </div>
+      <h3>Other</h3>
+      <mat-divider></mat-divider>
+      <div class="my-[10px]">
+        <a mat-button (click)="modelSourceSelection.emit('skip')">
+          <span> Manually configure model </span>
```

Comment:
`Manually configure model` isn't the right term for it. The button can be used to skip adding the model sources. Maybe something like: "Add the Model source later".

Reply:
Updated:
<img width="207" alt="ModelSourceSkipping" src="https://github.com/DSD-DBS/capella-collab-manager/assets/79602065/6ca20f46-7b8f-4383-b56b-857e2739e196">